external/VulkanMemoryAllocator/include/vk_mem_alloc.h

```diff
@@ -5686,8 +5686,7 @@ class VmaJsonWriter
     // Writes a number value.
     void WriteNumber(uint32_t n);
     void WriteNumber(uint64_t n);
-    void WriteSize(uint32_t n);
```

Comment:
Are you using a custom version of VMA for this?

Reply:
No, it's a revert of f7b95927db84c0bc3b6d6ad0f34f74276aa6c412
It doesn't seem like it broke any CIs but you may want to test some 32-bit platforms with it?